tests/acceptance/acceptancesuite.js

```diff
@@ -84,8 +97,8 @@ test.requestHooks(spellCheckLogger)('spell check flow', async t => {
 
   // Check that clicking spell check sends a queryTrigger=suggest url param
   // TODO(oshi) investigate making this an integration test
-  const requestUrl = spellCheckLogger.requests[spellCheckLogger.requests.length - 1].request.url;
-  const queryTriggerParam = new URLSearchParams(requestUrl).get('queryTrigger');
+  const queryParams = await getMostRecentQueryParamsFromLogger(spellCheckLogger);
```

Comment:
This doesn't feel much like an acceptance test. Are we essentially verifying that the `SpellCheck` component constructs the suggested query link properly? If so, I think we should aim to make this a unit or integration test.

Reply:
Yup! There's already a todo, I gave it an item (SLAP-1062) so it's a little easier to track